doc/release-notes/NEXT.md

```diff
@@ -7,12 +7,13 @@
 * Upgrade to .NET Framework 4.7.2 and upgrade NuGet dependencies (#1344 by @siprbaum)
 * Correct error message shown when `git tfs clone` has an error (#1347 by @siprbaum)
 * Add support for Visual Studio 2017 (no checkin pocicies yet). To use it set the
```

Comment:
We should remove `(no checkin pocicies yet)` (and just below also) because that's added in the last PR and so, from a "release note" point of view it is confusing for users....

Reply:
Ok, I will change that. Sorry that I merged this too quickly, as I figured it was uncontroversial as I only added the references to the PRs and the new entry